www/nginx/src/opnsense/www/js/nginx/dist/logviewer.min.js

```diff
@@ -1 +1 @@
et_template:function(){return"accesses"===this.type?h.a:"stream_accesses"===this.type?g.a:f.a}});const C=new(Backbone.View.extend({tagName:"div",className:"content-box tab-content",events:{"keyup .filter input":"update_filter","click #paging_first":"page_first","click #paging_back":"page_back","click #refresh":"update","click #paging_forward":"page_forward","click #paging_last":"page_last","change #entrycount":"change_entry_count"},page_entry_count:100,filter_delay:-1,initialize:function(){this.collection=new x,this.listenTo(this.collection,"sync",this.render),this.listenTo(this.collection,"update",this.render),this.listenTo(this.collection.filter_model,"change",this.render),this.type=""},render:function(){let e=this.$("tbody");e.length<1?0!==this.collection.length?(this.$el.html(y()({log_type:this.type,model:this.collection.filter_model})),e=this.$("tbody")):this.$el.html(j.a):e.html(""),0!==this.collection.length&&null==this.current_filtered_collection&&this.collection.forEach(t=>this.render_one(e,t)),this.$("#entrycountdisplay").html(this.page_entry_count),this.$("#currentpage").html(this.current_page+1),this.$("#pagecount").html(this.collection.page_count),this.$("#totalcount").html(this.collection.total_entries),this.$("#resultcount").html(this.collection.displayed_entries),this.current_page>=this.collection.page_count-1?(this.$("#paging_last").addClass("disabled"),this.$("#paging_forward").addClass("disabled")):(this.$("#paging_last").removeClass("disabled"),this.$("#paging_forward").removeClass("disabled")),this.current_page<=0?(this.$("#paging_back").addClass("disabled"),this.$("#paging_first").addClass("disabled")):(this.$("#paging_back").removeClass("disabled"),this.$("#paging_first").removeClass("disabled"))},render_one:function(e,t){const n=new T({type:this.type,model:t});n.render(),e.append(n.$el)},get_log:function(e,t,n){this.collection.uuid=t,this.collection.logType=e,this.collection.fileNo=n,this.type=e,this.current_page=0,this.$el.html(""),this.collection.filter_model.clear(),this.update()},update:function(){this.collection.page=this.current_page,this.collection.pageSize=this.page_entry_count,this.collection.fetch()},update_filter:function(e){clearTimeout(this.filter_delay);const t=e.target;this.collection.filter_model.set(t.name,$(t).val()),this.current_page=0,this.filter_delay=setTimeout(function(e){e.update()},500,this)},page_first:function(){this.current_page=0,this.update()},page_back:function(){this.current_page>0&&(this.current_page--,this.update())},page_forward:function(){this.current_page<this.collection.page_count&&(this.current_page++,this.update())},page_last:function(){this.current_page=this.collection.page_count-1,this.update()},change_entry_count:function(e){this.page_entry_count=e.target.value,this.current_page=0,this.update()}})),S=$("#logapplication").data("log"),q=new l({logType:S}),z=new p({collection:q,logview:C,logType:S});$("#logapplication").append(z.$el).append(C.$el),"global"!=S?q.fetch():z.render()}]);
```

Comment:
@8191 Hi! sorry to bother. but (if i understand it right) the source and minified versions not in sync now? can you please share a last logviewer source please? (i'm prepearing pr for the permanent ban requets logging). thanks!

Reply:
@8191 oops. sorry! I compared the wrong versions and raised unnecessary noise -  everything is relevant. sorry again!

Reply:
@kulikov-a What do you mean by _relevant_? Do I still need to look for the sources or is everything committed?

Reply:
@8191 Everything is fine: the source files correspond to the minified ones (was a bit confused by CRLFvsLF difference on builds). sorry again for the trouble!